src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/AddUserSteps.java

```diff
@@ -36,6 +36,7 @@ public void destroy() {
 
 	@Before(RunTest.HOOK.ADDUSER)
 	public void setLoginPage() {
+		System.out.println(".... User Login......");
```

Comment:
@gitcliff is this really the User Login Featue??
 please put the corresponding Feature name